src/api/handlers/team.py

```diff
@@ -58,6 +58,11 @@ def create(self, request):
 
     admin_emails = args.pop('admin_emails', '')
     args["admin_emails"] = parse_str_list(admin_emails)
+
+    parentId = args.pop('parent_id', None)
+    if parentId and parentId != 'undefined':
```

Comment:
this is great! Do you mind turning this into a helper function like `is_empty()` where it checks if its `undefined` `null` `None` etc